### DISCUSSION TITLE
Can you make user-submission forms in Quarto Pub websites?

### DISCUSSION THREAD

**Discussion by axiomtutor** [quarto-dev/quarto-cli#6551] [Q&A] [answered]
### Description

I just want to check if there's anything I missed in the documentation, about making buttons and forms and stuff.  I have a page hosted on Quarto Pub, and basically I have a bunch of exercise questions designed for a specific person.  I'd like to add submission forms to the page so that they can type and submit the answer right there.  (I don't need them logged in or anything -- there is a near-zero chance that anyone other than this person will ever see the site.)

Of course somehow the information has to get stored _somewhere_, if this is possible.  And it seems like Quarto Pub doesn't accommodate this -- which I think is probably understandable.  

I would guess that if I want this kind of functionality then I would need to host the site on something like Netlify or whatever, and use its server-side functionality and write the forms with raw HTML / CSS / JavaScript ... / PHP?.  But before going down that somewhat unfamiliar road I just wanted to check that there isn't anything Quarto built which would make the task easier.

**Comment by rgaiacs** [comment, marked as answer]
Quarto, like many other static web site generators, only support storage of data submitted by the user by using a third party service. A classical example is commenting, see https://quarto.org/docs/output-formats/html-basics.html#commenting.

You will have to implement a backend and implement the frontend form + submission.
